### PR TITLE
updated Clarinet README file

### PR DESCRIPTION
**Pull request by trichards-hiro**
Updated Clarinet README file with polish and some suggestions

Fixes https://github.com/hirosystems/clarinet/issues/478

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -733,38 +749,41 @@ $ clarinet integrate
...
-It's important any pull requests submitted have commit messages which follow this standard.
+This repo follows the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification when writing commit messages.
+
+**Note**It is important that any pull requests you submit have commit messages that follow this standard.
 
 To start contributing:
 
 1. Fork this repo and clone the fork locally.
-1. Create a new branch
+2. Create a new branch
```
Comment (hugocaillard) [inline README.md:774]: Markdown automatically handle the digits in front of ordered lists. We often use only `1.` everywhere and markdown will render `1, 2, 3 etc`. 
It makes it easier to rewrite / reorder

File: README.md [hunk 2]
```diff
@@ -733,38 +749,41 @@ $ clarinet integrate
...
    ```bash
    git checkout -b <my-branch>
    ```
-1. Run `npm i` in the local repo to install and initialize `husky` and `commitlint`.
+3. Run `npm i` in the local repo to install and initialize `husky` and `commitlint`.
```
Comment (hugocaillard) [inline README.md:778]: Same here `1.`,  and int the rest of this list

**Review by hugocaillard — COMMENTED** [review]
Great improvements. I made a few suggestions

**Review by sabbyanandan — CHANGES_REQUESTED** [review]
@trichards-hiro: Lots of great improvements! Thank you. 

The one meta-comment I have is: I find the README leeennngggthy; any ideas on how otherwise we could segment and structure so it is tight content-wise and easy for the readers?

Maybe the 'docs/how-to' kinds of README sections can be collapsed OFF by default (an idea of how to do that in MD is [here](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab)), and readers can expand to read specific sections? Even with sections collapsed, when someone does a browser search (ie: cmd+f), the collapsed sections will be included, too, anyway.

@obycode @lgalabru @hugocaillard: wdyt?

File: README.md [hunk 3]
```diff
@@ -1,43 +1,49 @@
 # Clarinet
 
 Clarinet is a Clarity runtime packaged as a command line tool, designed to facilitate smart contract understanding,
-development, testing and deployment. Clarinet consists of a Clarity REPL and a testing harness, which, when used
-together allow you to rapidly develop and test a Clarity smart contract, with the need to deploy the contract to a local
-devnet or testnet.
+development, testing and deployment. Clarinet consists of a Clarity Read-Evaluate-Print-Loop (REPL) environment and a testing harness. When used together, Clarity and the REPL environment enables you to rapidly develop and test a Clarity smart contract, allowing you to deploy the contract to a local devnet or testnet.
```
[the remark this answers is not in the document]
Comment (trichards-hiro) [reply README.md:4]: added definitions for the different environments

File: README.md [hunk 4]
```diff
@@ -57,27 +63,29 @@ xattr -d com.apple.quarantine /path/to/downloaded/clarinet/binary
 
 ### Install from source using Cargo
 
+If you would like to install Clarinet from source using Cargo, there are some specific steps you will need to follow, which are described below.
+
 #### Prerequisites
 
-[Install Rust](https://www.rust-lang.org/tools/install) for access to `cargo`, the Rust package manager.
+You must first [Install Rust](https://www.rust-lang.org/tools/install) to gain access to cargo, which is the Rust package manager.
```
[the remark this answers is not in the document]
Comment (trichards-hiro) [reply README.md:70]: reworded this to make more clear

File: README.md (hunk 1 above)
Comment (trichards-hiro) [reply README.md:779]: fixed

File: README.md [hunk 5]
```diff
@@ -1,43 +1,49 @@
...
 ![screenshot](docs/images/demo.gif)
 
 ### Clarinet 101
 
-An introductory video tutorial series, from Hiro Engineer [Ludo Galabru](https://twitter.com/ludovic?lang=en), that guides developers through some of the fundamentals of of Clarinet, and how it can help develop, test, and deploy Clarity smart contracts.
+Hiro has created an introductory video tutorial series that guides you through some of the fundamentals of of Clarinet, and how it can help develop, test, and deploy Clarity smart contracts.
 
-Check out the playlist on [Hiro's Youtube](https://www.youtube.com/c/HiroSystems):
-[<img src="docs/images/clarinet101.png">](https://youtube.com/playlist?list=PL5Ujm489LoJaAz9kUJm8lYUWdGJ2AnQTb)
+Please revew and watch the YouTube playlist on [Hiro's Youtube](https://www.youtube.com/c/HiroSystems):
```
[the remark this answers is not in the document]
Comment (lgalabru) [reply README.md:20]: Should we limit the number of call to actions? Either Hiro's playlist, or Clarinet's playlist (the latter imo)

**Review by lgalabru — APPROVED** [review]
@sabbyanandan 
> I find the README leeennngggthy

100% agree. I would also be happy merging this PR (doing a great job on the language) as iteration #1, and tackle the length issue in a subsequent PR.

**Comment by sabbyanandan** [comment]
> I would also be happy merging this PR (doing a great job on the language) as iteration https://github.com/hirosystems/clarinet/issues/1, and tackle the length issue in a subsequent PR.

@lgalabru: Yes! And we will get to https://github.com/hirosystems/clarinet/issues/596 when @trichards-hiro returns from PTO. Other than this, let's go ahead with these changes for now. 👍🏽

File: README.md (hunk 2 above)
Comment (trichards-hiro) [reply README.md:783]: fixed

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (trichards-hiro) [reply README.md:20]: fixed in document so we only make one call to action for the Clarinet playlist

**Comment by trichards-hiro** [comment]
hi @hugocaillard and @lgalabru : I have pushed some of my changes so I wanted to ask you both if there is anything else I should do before we push these final changes.

Thank You :)

**Review by hugocaillard — APPROVED** [review]
LGTM 👍

**Review by sabbyanandan — APPROVED** [review]
Let's go!

**Pull request merged**